src/ui/pages/Menu/components/ConnectWallet/ConnectWallet.scss

```diff
@@ -10,6 +10,14 @@
     height: calc(100vh - 8rem);
   }
 
+  .card-list {
```

Comment:
Ah I thought this component was shared - can we apply this to the other KERI Connections list too in this PR if that's OK? Not sure if this happens in any other place.

Reply:
Yep. I applied `CardList` component for Keri Connection. Please help me check:
 ### IOS
 
<img width="1101" alt="Screenshot 2024-06-06 at 17 15 19" src="https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/2eebec50-7f00-449f-9980-df23befed475">

 ### Android
 
![Screenshot 2024-06-06 at 17 25 43](https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/7a4bdd64-0dd5-4ef7-b34b-9c0cfd1ad685)

 ### Web
![Screenshot 2024-06-06 at 17 07 02](https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/3c528ae9-60c3-4f74-81d8-4f14d88ae606)

Reply:
Ah I actually thought there was no line at the end of this list too, which was my mistake. But looks like a nice refactor